scripts/deploy.sh

```diff
@@ -2,10 +2,15 @@
 
 set -e
 
+HERE="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+
 source "${HOME}/.cargo/env"
 source "${HOME}/.nvm/nvm.sh"
 nvm use
 
+echo "Setting rustup override for this project"
+rustup override set `cat rust-toolchain`
```

Comment:
Please, don't use backticks. Use `$()` as it is POSIX & can be nested (copy & pasted freely around). And be consistent with `HERE="$...`.